### DISCUSSION TITLE
How to Ignore Time Zones on the Employee Check-In Check-Out template.

### DISCUSSION THREAD

**Discussion by Srij1** [Budibase/budibase#11968] [Help] [answered]
Hey, I was using this template for a project, but whenever I check someone in or out, it is done in GMT. I wish to change this to CDT or just ignore the time zone and use the one on the date picker. Thanks for your help and Budibase has been working great for me!

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Srij1,

Which data source are you using?

Have you tried checking the ignore timezones toggle on both the date picker and the date/time field within Budibase?

<img width="343" alt="Screenshot 2023-10-04 at 02 50 34" src="https://github.com/Budibase/budibase/assets/126772285/24bd0508-2092-4bac-a414-41da6b5329ad">
<img width="630" alt="Screenshot 2023-10-04 at 02 55 07" src="https://github.com/Budibase/budibase/assets/126772285/e602d1cd-861a-4239-aae1-03139b061a37">

Let me know if this works for you.

If you are self-hosting this [thread](https://github.com/Budibase/budibase/discussions/11729) may also help you.

**Reply by Srij1** [reply]
Yeah, this worked well, thank you!
